<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchBar</class>
 <widget class="QWidget" name="SearchBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>34</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolButton" name="closeButton">
     <property name="toolTip">
      <string>Close the search bar</string>
     </property>
     <property name="icon">
      <iconset theme="dialog-close"/>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="searchInfo">
     <property name="text">
      <string comment="label for input line to find text">&amp;Find:</string>
     </property>
     <property name="buddy">
      <cstring>searchComboBox</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KHistoryComboBox" name="searchComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="nextButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Find the next match for the current search phrase</string>
     </property>
     <property name="text">
      <string>&amp;Next</string>
     </property>
     <property name="icon">
      <iconset theme="go-down-search"/>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="previousButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Find the previous match for the current search phrase</string>
     </property>
     <property name="text">
      <string>&amp;Previous</string>
     </property>
     <property name="icon">
      <iconset theme="go-up-search"/>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="optionsButton">
     <property name="toolTip">
      <string>Find the previous match for the current search phrase</string>
     </property>
     <property name="text">
      <string>&amp;Options</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionMatchCase">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Match Case</string>
   </property>
  </action>
  <action name="actionSearchAutomatically">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Search As You Type</string>
   </property>
  </action>
  <action name="actionHighlightMatch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Highlight All Matches</string>
   </property>
   <property name="toolTip">
    <string>Highlight Matches</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>SearchBar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
